doc/connectivity/bluetooth/api/mesh/proxy.rst

```diff
@@ -9,6 +9,72 @@ mesh network through GATT. The Proxy feature is only compiled in if the
 controlled by the :ref:`bluetooth_mesh_models_cfg_srv`, and the initial value
 can be set with :c:member:`bt_mesh_cfg_srv.gatt_proxy`.
 
+Nodes with the Proxy feature enabled can advertise with Network Identity and Node Identity,
+which is controlled by the :ref:`bluetooth_mesh_models_cfg_cli`.
+
+The GATT Proxy state indicates if the Proxy feature is supported.
+
+Private Proxy
+*************
+
+A node supporting the :ref:`bluetooth_mesh_models_priv_beacon_srv` can advertise with
+Private Network Identity and Private Node Identity types, which is controlled by the
+:ref:`bluetooth_mesh_models_priv_beacon_cli`. Private Beacons provide privacy by periodically
+randomizing the beacon input data. By advertising with this set of identification types,
```

Comment:
Sentence `Private Beacons provide privacy by periodically randomizing the beacon input data.` is belongs to Private Beacons, but here we don't send beacons. The next sentence explains the need for Private types so I think this sentence can be removed.